policy-controller/k8s/index/src/inbound/index.rs

```diff
@@ -1753,6 +1753,12 @@ impl PolicyIndex {
             authzs.insert(reference, authz);
         }
 
+        if let Some(p @ DefaultPolicy::Allow { .. }) | Some(p @ DefaultPolicy::Audit) =
```

Comment:
do we need the complex match condition here?  can this be `if let Some(p) = server.access_policy { authzs.extend(p.default_authzs(&self.cluster_info)) }` ?

This should be equivalent because DefaultPolicy::Deny should add no authorizations, right?

Reply:
Good catch :+1: